src/index.js

```diff
@@ -93,9 +93,9 @@ class OverledgerSDK {
    * @param {Object} signedTransactions Object of the DLTs where you want to send a transaction
    */
   send(signedTransactions) {
-    const apiCall = signedTransactions.map((dlt) => {
-      return this.dlts[dlt.dlt].buildApiCall(dlt.signedTransaction);
-    });
+    const apiCall = signedTransactions.map(
+      dlt => this.dlts[dlt.dlt].buildApiCall(dlt.signedTransaction),
```

Comment:
The extra comma should be avoided in a function declaration when not used. 
```suggestion
      dlt => this.dlts[dlt.dlt].buildApiCall(dlt.signedTransaction)
```